src/main/scss/overlay.scss

```diff
@@ -132,7 +141,7 @@ $overlay-headline-height: 50px;
       margin: 0;
       border-radius: 2em;
       cursor: pointer;
-      background-color: #EFEFEF;
+      background-color: white;
```

Comment:
It's a temporary replacement approeved by Olli, as he's working on this style atm.